Concrete role types (PR 6885) (#714)
graknlabs/docs: typeql-src/modules/ROOT/pages/schema/define-types.adoc

[,typeql]
----

> define

Author: Just fixing indentation and formatting for TypeQL on this page

----
define

access sub relation,
    relates object;

object sub entity,
    abstract,
    plays access:object;
----

==== Subtypes another entity

[NOTE]
====
All types that are subtyping `entity` built-in type directly or through other subtypes are called entity types.

> Instances of these types are called entities. The same approach can be applied to attributes and relations.

Reviewer: Should "Approach" be "terminology"? -- eg "Attributes and relations use analogous terminology". Or something?

Is it "can" or "is/are/should be" (more assertive?)

Author: ```suggestion
Instances of entity types are called entities. A similar terminology is applied to attributes and relations.
```
